### ISSUE TITLE
Licensing Question

### ISSUE THREAD

**Issue by mike-hanson** [spazzarama/SharedMemory#72]
Can I use the NuGet package in a closed source application without violating the open source MIT License terms?

**Comment by daxpandhi** [comment]
Yes, you can as long as you include the copy of the license text and credit as described in https://github.com/justinstenning/SharedMemory/blob/master/LICENSE.md

Somewhere in the about box, or a text file distributed with your application should satisfy the requirements.

**Comment by mike-hanson** [comment]
Thanks Justin.
